cargo-miri/bin.rs

```diff
@@ -495,17 +520,26 @@ fn phase_cargo_miri(mut args: env::Args) {
     }
     cmd.env("RUSTC_WRAPPER", &cargo_miri_path);
 
-    // Set the runner for the current target to us as well, so we can interpret the binaries.
-    let runner_env_name = format!("CARGO_TARGET_{}_RUNNER", target.to_uppercase().replace('-', "_"));
-    cmd.env(&runner_env_name, &cargo_miri_path);
+    // Set the runners to us as well, so we can interpret the binaries or skip unit tests of
+    // `proc-macro` crates.
```

Comment:
Please extend the comment to explain why both host and target runner need to be set here.